package.json

```diff
@@ -133,6 +133,7 @@
     "fastify": "1.14.6",
     "fastify2": "npm:fastify@2.15.3",
     "fastify3": "npm:fastify@3.21.5",
+    "fastify4": "npm:fastify@4.24.3",
```

Comment:
If there is no good reason to test against four major versions, could you please drop the last two major versions?
And: could you please use the v4 as latest version?